js/index.js

```diff
@@ -34,12 +34,24 @@ function submitForm(event) {
     removeButton.innerText = "remove";
     removeButton.type = "button";
     removeButton.addEventListener('click', removeEvent);
-    function removeEvent() {
-    let entry = document.querySelector('button').parentNode;
-    entry.remove();
+    function removeEvent(e) {
+      e.target.parentNode.remove()
     };
     newMessage.appendChild(removeButton);
     messageList.appendChild(newMessage);
     messageForm.reset();
 }
 
+window.onscroll = function() {myFunction()};
+
+var header = document.getElementById("myHeader");
```

Comment:
try to use `let` or `const` instead of `var`

Reply:
ok